docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -0,0 +1,150 @@
+lrwxrwxrwx. 1 yugabyte yugabyte   85 Jan 15 19:28 linuxbrew-xxxxxxxxxxxx -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/linuxbrew-xxxxxxxxxxxx
+lrwxrwxrwx. 1 yugabyte yugabyte   71 Jan 15 19:28 postgres -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/postgres
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 pylib -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/pylib
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 share -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/share
+lrwxrwxrwx. 1 yugabyte yugabyte   68 Jan 15 19:28 tools -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/tools
+lrwxrwxrwx. 1 yugabyte yugabyte   65 Jan 15 19:28 ui -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/ui
+lrwxrwxrwx. 1 yugabyte yugabyte   84 Jan 15 19:28 version_metadata.json -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/version_metadata.json
+lrwxrwxrwx. 1 yugabyte yugabyte   66 Jan 15 19:28 www -> /home/yugabyte/yb-software/yugabyte-2.5.1.0-b187-centos-x86_64/www
+-rw-rw-r--. 1 yugabyte yugabyte    0 Jan 15 19:29 master.out
+-rw-rw-r--. 1 yugabyte yugabyte 2200 Jan 15 20:18 master.err
+drwxr-xr-x. 2 yugabyte yugabyte   25 Jan 15 20:18 conf
+```
+
+### Config Files
+You can find config files for the master and tserver at `/home/yugabyte/{master|tserver}/conf/server.conf`.
+
+### Transaction Logs a.k.a WAL (write-ahead logs)
+You can find WAL files at `/mnt/d*/yb-data/{master|tserver}/wals`.
+
+To pretty print the contents of the WAL files, use the `log-dump` utility:
+
+```
+$ pwd
+/home/yugabyte
+
+$ ./tserver/bin/log-dump /mnt/d0/yb-data/tserver/wals/table-e85a116bc557403e82f57037e7b13879/tablet-05bef5ed6fb74cabb420b648b6f850e3/
+
+# use -print_entries=pb to print the entire contents of each record
+```
+
+### Database (SSTable) Files for various tablets
+You can find SSTable files at `/mnt/d*/yb-data/{master|tserver}/data`.
+
+To pretty print the contents of these SSTable files, 
+
+```
+$ pwd
+/home/yugabyte
+
+$ ./tserver/bin/ldb dump --compression_type=snappy --db=/mnt/d0/yb-data/tserver/wals/table-e85a116bc557403e82f57037e7b13879/tablet-05bef5ed6fb74cabb420b648b6f850e3/
+```
+
+### Debug Logs (.INFO, .WARNING, .FATAL, etc.)
+Debug logs are output to `/home/yugabyte/{master|tserver}/logs`.
+
+### View Standard Output/Standard Error (stderr/stdout) of yb-tserver & yb-master process
+
+These go to e.g. `/home/yugabyte/tserver/tserver.{err|out}.
+
+## Setting gflags dynamically
+
+The `yb-ts-cli` utility allows changing gflags dynamically. _Note: For this step you must identify the server using its RPC port (not the HTTP port).
+
+For example, to increase the verbose logging level to 2:
+
+    $ ./yb-ts-cli --server_address=localhost:9100 set_flag v 2
+
+Or, to turn tracing on for all RPCs:
+
+    $ ./yb-ts-cli --server_address=localhost:9100 set_flag rpc_dump_all_traces 1
```

Comment:
should we mention that some of the flags require you to use -force?

Reply:
Also, common question is whether vmodule can be set at runtime. This cannot be set at runtime. only during start

Reply:
@acube123 which flags require -force and what are the semantics of that?

Reply:
Basically any flag that is not explictly marked TAG_FLAG(xxx, runtime) will need a -force flag.

Ideally, we should be updating the flags that are *safe* to be updated on the fly -- some that use GetAtomicFlag(&) and fetch the value for each use rather than using the flag to initialize an object once; as runtime; so that no one needs to use the -force flag. We are not there yet. So, may need the -force flags for flags that are not safe to update ... but not yet marked as runtime.